index.d.ts

```diff
@@ -204,6 +204,26 @@ export interface CommonOptions<EncodingType> {
 	*/
 	readonly killSignal?: string | number;
 
+	/**
+	You can abort the spawned process using [`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController).
+
+	*Requires Node.js 16 or later.*
+
+	@example
+	```
+	import {execa} from 'execa';
+
+	const abortController = new AbortController();
+	const subprocess = execa('node', [], {signal: abortController.signal});
+
+	abortController.abort();
+
+	console.log(subprocess.killed);
+	//=> true
+	```
+	*/
```

Comment:
The same docs should be in the readme too.

Reply:
Could we add an example [here](https://github.com/sindresorhus/execa/pull/490/commits/7ed6580e7ed657be1d7e16349ec85f17777ba2e0#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R567) link to avoid document duplication?